index.js

```diff
@@ -24,7 +24,7 @@ const defaults = {
   d3Module: require('d3'), // to allow use of d3.v4
   selector: '',  // selects base D3 Element
   container: '', // markup inserted in body
-  svgStyles: ''  // embedded svg stylesheets
+  styles: ''  // embedded svg stylesheets
```

Comment:
@geekplux this is a breaking API change. You should create an alias for this prop first.